vg/test_helpers.py

```diff
@@ -1,2 +1,2 @@
 def get_imported_names(module):
-    return [name for name in dir(module) if not name.startswith("_")]
+    return sorted([name for name in module.__all__ if not name.startswith("_")])
```

Comment:
Is anything with underscores being exported?

Reply:
No

Reply:
Then this is equivalent to `sorted(module.__all__)`

Reply:
updated in https://github.com/lace/vg/pull/232

Reply:
thanks!